<launch>
    <arg name="ns" default="" />
    <node pkg="create_node" type="turtlebot_node.py" name="icreate_node" output="screen">
        <!--<param name="port" type="string" value="/dev/ttyUSB0"/>-->
        <param name="port" type="string" value="/dev/ttyUSB1"/>
	    <param name="publish_tf" type="string" value="False"/>
        <param name="cmd_vel_timeout" type="int" value="10"/>
        <remap from="/diagnostics" to="diagnostics" />
        <param name="odom_frame" value="$(arg ns)/odom"/>
        <param name="base_frame" value="$(arg ns)/base_footprint"/>
        <remap from="imu/data" to="icreate_node/imu/data" />
        <remap from="imu/raw" to="icreate_node/imu/raw" />
        <!--<param name="odom_frame" value="odom"/>-->
        <!--<param name="base_frame" value="base_footprint"/>-->
        
    </node>
</launch>